### PR TITLE
chore(sample): add one productivity hub sample

### PR DESCRIPTION
**Pull request by SLdragon**
graph-toolkit-one-productivity-hub

### CODE REVIEW

File: packages/fx-core/src/common/samples-config.json [hunk 1]
```diff
@@ -42,6 +42,16 @@
...
         },
+        {
+            "id": "graph-toolkit-one-productivity-hub",
+            "title": "One Productivity Hub using Graph Toolkit",
+            "shortDescription": "One Productivity Hub app of Microsoft Teams Tab app.",
+            "fullDescription": "One Productivity Hub sample shows you how to build a tab for viewing your calendar events, to-do tasks and files by using Microsoft Graph Toolkit as UI component.",
+            "tags": ["Tab", "JS"],
+            "time": "5min to run",
+            "configuration": "Ready for local debug",
+            "suggested": true
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/common/samples-config.json:53]: Is this a suggested sample by PM/UX?

File: packages/fx-core/src/common/samples-config.json [hunk 2]
```diff
@@ -42,6 +42,16 @@
             "configuration": "Ready for local debug",
             "suggested": true
         },
+        {
+            "id": "graph-toolkit-one-productivity-hub",
+            "title": "One Productivity Hub using Graph Toolkit",
+            "shortDescription": "One Productivity Hub app of Microsoft Teams Tab app.",
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/common/samples-config.json:48]: This seems not exactly the same with what Ayca provided, is this updated version?

**Comment by HuihuiWu-Microsoft** [comment]
Can you help attach one gif on the responsive layout in sample gallery to make sure it works? Thanks!

File: packages/fx-core/src/common/samples-config.json (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/src/common/samples-config.json:53]: @zyxiaoyuer , what do think about this?

File: packages/fx-core/src/common/samples-config.json (hunk 2 above)
Comment (SLdragon) [reply packages/fx-core/src/common/samples-config.json:48]: I follow the decryption of "graph toolkit contact exporter" sample.

**Comment by SLdragon** [comment]
> Can you help attach one gif on the responsive layout in sample gallery to make sure it works? Thanks!

Tested works fine

File: packages/fx-core/src/common/samples-config.json (hunk 1 above)
Comment (zyxiaoyuer) [reply packages/fx-core/src/common/samples-config.json:53]: by Suggested, do you mean the star? 
BTW. I remeber there is a SSO tag also.

File: packages/fx-core/src/common/samples-config.json (hunk 2 above)
Comment (zyxiaoyuer) [reply packages/fx-core/src/common/samples-config.json:48]: 1. Description short: Microsoft Teams tab with SSO to view calendar events, to-do tasks and file folders.
2. Description long: One Productivity Hub sample shows you how to build a tab for viewing your calendar events, to-do tasks and files by using Microsoft Graph Toolkit components and TeamsFx provider. You can run this app locally or deploy it to Microsoft Azure.****

Comment (SLdragon) [reply packages/fx-core/src/common/samples-config.json]: thanks, updated

File: packages/fx-core/src/common/samples-config.json (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/src/common/samples-config.json]: Thanks, updated suggested to false, and add sso tag

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

**Review by zyxiaoyuer — APPROVED** [review]

**Pull request merged**